stable/anchore-engine/templates/enterprise_feeds_configmap_env.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if and .Values.anchoreEnterpriseGlobal.enabled (or .Values.anchoreEnterpriseRbac.enabled .Values.anchoreEnterpriseReports.enabled) -}}
+{{- $component := "enterprise" -}}
```

Comment:
the component for this is "enterprise-feeds"

Reply:
Thanks for catching those :+1: